src/objects/Hierarchy_Settings__c.object

```diff
@@ -95,28 +95,28 @@
     <fields>
         <fullName>Affl_ProgEnroll_Copy_End_Date__c</fullName>
         <defaultValue>false</defaultValue>
-        <description>Indicates whether to set the End Date of the Affiliation created with a Program Enrollment. If enabled, set the End Date to the Program Enrollment&apos;s End Date. If disabled, don&apos;t specify the Affiliation&apos;s End Date.</description>
+        <description>Indicates whether to set the End Date of the Affiliation created with a Program Enrollment. If enabled, set the End Date to the Program Enrollment&apos;s End Date. If disabled, don't specify the Affiliation&apos;s End Date.</description>
```

Comment:
@xinnanlisf -- I remember these coming across in a different PR and we had replaced the individual apostrophe marks with the encoded &apos; value instead.  I see that here we appear to be putting them back to the individual apostrophe mark. Is there a reason for this? if you look at these lines, it appears that for we aren't being consistent. We have Enrollment&apos;s for "Enrollment's" and we have "don't" later on in the text without the &apos; encoded version

Reply:
Hi @lifewithryan, yeah I saw that this morning too and I believe the changes have been reverted too ;)